dev/RuntimeCompatibilityOptions/RuntimeCompatibilityOptions.idl

```diff
@@ -33,6 +33,10 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         ApplicationDataProvider_ReentrancyProtection = 57688028,
         InputPointerSource_FlushReentrancyCrash = 57705537,
         WindowChrome_SetTitleBarCrash = 57693465,
+
+        // 1.7.4
+        DispatcherQueue_PropagateException = 58472221,
```

Comment:
Nit: I've been keeping each group sorted alphabetically, but obviously that doesn't really matter.